gulpfile.babel.js

```diff
@@ -88,4 +88,10 @@ gulp.task('bootstrap_test', () => {
     .once('end', () => {
 	process.exit();	
     })
-});
\ No newline at end of file
+});
+
+gulp.task('clean', () => {
+  gulp.src(['out/'], {read: false})
```

Comment:
I'd happy to add more things such as node_modules, .pkg_timestamp and so on.
If developer uses this command, should go back to initial status.